src/Infrastructure/Builder/Yaml/LifecycleBuilderInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Builder\Yaml;
+
+use SprykerSdk\SdkContracts\Entity\Lifecycle\TaskLifecycleInterface;
+
+interface LifecycleBuilderInterface
```

Comment:
If you use DTO, you can move interface to the application layer then and combine them under single interface. Moving it to the application layer also fits onion architecture because the dependency goes inside

Reply:
But what about return types? Return types differ between builders